server-profiles/11.3.0/README.md

```diff
@@ -0,0 +1,8 @@
+# Purpose
```

Comment:
What's the difference between 11.3.0 and 11.3.4 profiles?

Reply:
11.2.2 and 11.2.7 are the same and are the only versions that have a different data.json.subst file. The multiple version folders are mainly just to simplify everything using the existing `PINGFEDERATE_PROVIDER_PRODUCT_VERSION` environment variable.

Reply:
I'd like to avoid the fully duplicated profiles. Possibly with a separate variable for the profile version, or maybe it's possible by mounting individual files

Reply:
agreed

Reply:
added shared profile with individual version data.json.subst files